frontend/sass/pages/_movies.scss

```diff
@@ -222,6 +213,10 @@
       flex-wrap: wrap;
       gap: 10px;
     }
+
+    .movie-info h2 {
+      font-size: 3em;
```

Comment:
`.movie-info h2` määrritellään nyt kahdessa paikassa voisiko ne yhdistää

Reply:
On määritelty kahdessa paikkaa, koska ensimmäinen fiksaa line-heightin kaikille ruutuleveyksille (pitkät nimet näytti hassulta useammalla rivillä koska rivien välissä oli paljon tilaa), toinen taas pienentää tekstin kokoa kapealla näytöllä että varsinkin pitkät nimet asettuu vähän nätimmin. Jos puhutaan siis tämän tiedoston sisässä olevista h2-muotoiluista.

Reply:
Joo puhutaan tämän tiedoston sisäisitä määrittelyistä. Ne varmaan voisi yhdistää yhdeksi määrittelyksi

Reply:
Siirsin molemmat nyt movie-infon sisään, en oo ihan varma oliko se nyt se muutos mitä haluttiin, toivottavasti oli 😄

Reply:
Ääh en huomannut tuota `@media only screen and (max-width: abstracts.$large-width) {` palauta se alkuperäiseen muotoon se oli parempi. Anteeksi sekoiluni

Reply:
Ei mitään, mää palautan! 😄